app/src/main/res/layout/activity_main.xml

```diff
@@ -6,14 +6,13 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
-    <TextView
-        android:id="@+id/helloAndroid"
-        android:layout_width="wrap_content"
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/factsRecyclerView"
+        android:layout_width="match_parent"
```

Comment:
As I remember, you have to use `0dp` except of `match_parent` if you set constraints (`app:layout_constraintStart_toStartOf`, `layout_constraintEnd_toEndOf`). The same with the height. Android Studio should highlight this value and suggest to change it.

Reply:
FrameLayout is good here too.